### PR TITLE
Init react admin + vite

### PR DESCRIPTION
**Pull request by atilbian**
- [x] Init react admin project with vite
- [x] Update docker-compose.dev

Postponed :
- Use yarn workspace and shared depencies inside our docker-compose dev env
- Use yarn workspace to share depencies between our apps

Related to https://trello.com/c/HSUZcSyK/37-3-as-norbert-i-want-to-be-able-to-see-all-games-and-be-able-to-delete-some

### CODE REVIEW

File: admin-app/src/App.css [hunk 1]
```diff
@@ -0,0 +1,42 @@
+.App {
```
Comment (djhi) [inline admin-app/src/App.css]: I think you can safely remove this file

File: admin-app/src/index.css [hunk 2]
```diff
@@ -0,0 +1,13 @@
+body {
```
Comment (djhi) [inline admin-app/src/index.css]: Same

File: admin-app/src/logo.svg [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 841.9 595.3">
```
Comment (djhi) [inline admin-app/src/logo.svg]: Same

File: admin-app/src/App.css (hunk 1 above)
Comment (atilbian) [reply admin-app/src/App.css]: Ok thx, it's done !

**Review by djhi — APPROVED** [review]

**Review by slax57 — APPROVED** [review]

**Pull request merged**